kiwi/solverimpl.h

```diff
@@ -382,24 +383,22 @@ class SolverImpl
 	for tracking the movement of the constraint in the tableau.
 
 	*/
-	Row* createRow( const Constraint& constraint, Tag& tag )
+	std::unique_ptr<Row> createRow( const Constraint& constraint, Tag& tag )
```

Comment:
What is the advantage of using a unique_ptr here ?

Reply:
It makes it exception-safe by not letting the raw pointer a chance to escape.

Reply:
Thanks for the explanation.